modules/payment/src/PaymentMethodListBuilder.php

```diff
@@ -4,6 +4,7 @@
 
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Entity\EntityListBuilder;
+use Drupal\Core\Routing\RouteMatchInterface;
```

Comment:
Is this use still needed now that the `@var` no longer uses the imported interface?